js/private/js_binary.bzl

```diff
@@ -259,6 +259,16 @@ _ATTRS = {
         """,
         default = True,
     ),
+    "optimized_runfiles_collection": attr.bool(
```

Comment:
How about "include legacy runfiles" and `False` by default? Something to make it sound old/bad/deprecated...

Reply:
Perhaps... tho "legacy" doesn't quite describe it in my mind. It is an overbroad, defence-in-depth runfiles collection mode that some users may need to use if they are using deps from other rule sets or custom rules that don't define their runfiles correctly.

Reply:
I guess I mean to to highlight the "don't define their runfiles correctly" part of it...